articles/libraries/lock/v10/migration-guide.md

```diff
@@ -4,39 +4,74 @@ Heads up! This document is using the latest version of Lock (version 10). See ch
+  - The `disableResetAction` option was renamed to [allowForgotPassword](/libraries/lock/v10/customization#allowforgotpassword-boolean-).
+  - The `disableSignUpAction` option was renamed to [allowSignUp](/libraries/lock/v10/customization#allowsignup-boolean-).
+  - The `defaultUserPasswordConnection` option has been replaced by the [defaultDatabaseConnection](/libraries/lock/v10/customization#defaultdatabaseconnection-string-) and the [defaultEnterpriseConnection](/libraries/lock/v10/customization#defaultenterpriseconnection-string-) options.
+  - The `resetLink` option was renamed to [forgotPasswordLink](/libraries/lock/v10/customization#forgotpasswordlink-string-).
+
+### Other Options
   - The `forceJSONP` option was removed.
-  - The `gravatar` option was renamed to `avatar` and instead of taking `true` and `false` it now takes `null` or an object. See the [New Features page](/libraries/lock/v10/new-features#custom-avatar-provider) for details.
-  - The `icon` option was renamed to `logo` and namespaced under `theme`. Now you use it like this `theme: {logo: "https://example.com/icon.png"}`.
-  - The `popup` option was replaced by `redirect` which is namespaced under `auth`. If you previously used `popup: true` now you need to provide `auth: {redirect: false}`.
-  - The `primaryColor` option was namespaced under `theme`. Now you use it like this `theme: {primaryColor: "#ec4889"}`.
-  - The `resetLink` option was renamed to `forgotPasswordLink`.
-  - The `responseType` option was namespaced under `auth`.  Now you use it like this `auth: {responseType: "code"}`.
-  - The `socialBigButtons` option was renamed to `socialButtonStyle` and its possible values are `"small"` or `"big"` instead of `true` or `false`.
-  - The `sso` option was namespaced under `auth`.  Now you use it like this `auth: {sso: false}`.
-- Some other options were added, see [New Features page](/libraries/lock/v10/new-features) for details.
 
+---
+
+## Further Reading
+  - Some other options were added, see [New Features page](/libraries/lock/v10/new-features) for details.
+  - Check out the [customization page](/libraries/lock/v10/customization) for more details on all of the customization options that are available.
+  - Take a look at the [api page](/libraries/lock/v10/api) for more details on Lock 10's API.
 
-## Upgrading Between (or From) Preview Releases
+## Upgrading From Preview Releases
 
-This is a summary of what you absolutely need to know before upgrading between preview releases. For the full list of changes, please see the project's [CHANGELOG](https://github.com/auth0/lock/blob/v10/CHANGELOG.md).
+**This section is only pertinent if you are currently using a pre-release version of Lock 10, and wish to update to the release version.** It is a summary of what you absolutely need to know before upgrading between preview releases. For the full list of changes, please see the project's [CHANGELOG](https://github.com/auth0/lock/blob/v10/CHANGELOG.md).

```

Comment:
@jeffreylees The [CHANGELOG](https://github.com/auth0/lock/blob/v10/CHANGELOG.md) URL is broken.